ts/data/sample.ts

```diff
@@ -858,31 +906,40 @@ function addPhixContents(
     metrics.forEach((metric) => addMetricRequirementText(metric, fragment));
   };
 
+  const minPhixValue = Math.min(
+    ...sample.run.lanes
+      .flatMap((lane) => [lane.percentPfixRead1, lane.percentPfixRead2])
+      .filter((value) => typeof value === "number")
+  );
+
+  const contentWrapper = document.createElement("div");
   const multipleLanes = sample.run.lanes.length > 1;
+
   sample.run.lanes.forEach((lane) => {
     const laneDiv = document.createElement("div");
     laneDiv.classList.add("whitespace-nowrap", "print-hanging");
-    let text = multipleLanes ? `L${lane.laneNumber}` : "";
-    if (nullOrUndefined(lane.percentPfixRead1)) {
-      const span = document.createElement("span");
-      span.innerText = text + ": ";
-      laneDiv.appendChild(span);
-      laneDiv.appendChild(makeNotFoundIcon());
-    } else {
-      if (multipleLanes) {
-        text += " ";
-      }
-      text += `R1: ${lane.percentPfixRead1}`;
-      if (!nullOrUndefined(lane.percentPfixRead2)) {
-        text += `; R2: ${lane.percentPfixRead2}`;
-      }
-      laneDiv.innerText = text;
-      if (addTooltip) {
-        tooltip.addTarget(laneDiv, addContents);
-      }
+
+    const laneText = multipleLanes ? `L${lane.laneNumber}: ` : "";
+    laneDiv.innerHTML = nullOrUndefined(lane.percentPfixRead1)
+      ? `${laneText}<span>${makeNotFoundIcon()}</span>`
+      : `${laneText}R1: ${lane.percentPfixRead1}${
+          !nullOrUndefined(lane.percentPfixRead2)
+            ? `; R2: ${lane.percentPfixRead2}`
+            : ""
+        }`;
+
+    if (addTooltip && !nullOrUndefined(lane.percentPfixRead1)) {
+      tooltip.addTarget(laneDiv, addContents);
```

Comment:
I don't see why there was any need to rewrite this, but it's now less readable, and also broken if any PhiX values are null:

<img width="177" alt="image" src="https://github.com/oicr-gsi/dimsum/assets/12240540/f0656d57-7b1a-4e29-9b7b-f25c539b8c7e">